pom.xml

```diff
@@ -144,11 +144,28 @@
                 <artifactId>nifi-record-sink-api</artifactId>
                 <version>${nifi.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.nifi</groupId>
+                <artifactId>nifi-standard-record-utils</artifactId>
+                <version>${nifi.version}</version>
+            </dependency>
             <dependency>
                 <groupId>org.apache.nifi</groupId>
                 <artifactId>nifi-schema-registry-service-api</artifactId>
                 <version>${nifi.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.nifi</groupId>
+                <artifactId>nifi-record-serialization-services</artifactId>
+                <version>${nifi.version}</version>
+                <scope>test</scope>
```

Comment:
we should double check how the scope of these impacts things - nifi processors are not entirely intuitive with the expected scope of dependencies in a nar.

Reply:
Ok, thoughts about how to do that well?

Reply:
i was thinking just compare the dependency tree from before and after and make sure none of the existing dependencies changed scope